### ISSUE TITLE
[BottomSheet] In voice over, the background tap is not accessible

### ISSUE THREAD

**Issue by jverkoey** [material-components/material-components-ios#4207]
> The only way to dismiss the bottom sheet is to swipe up, which is not very intuitive.
> 
> (We also provide a cancel button, but the dimmed, background area should be reachable via Voice Over)

## Reproduction steps

1. Enable VoiceOver.
2. Go to the bottom sheet demo in MDCCatalog.
3. Press the button to present the bottom sheet.
4. Attempt to tap the background in order to dismiss the bottom sheet.

## Expected behavior

The background can be selected and double-tapped in order to dismiss the bottom sheet.

## Actual behavior

The background is not tappable.

---

If you are a Googler, please visit [b/79995984](http://b/79995984) for more details.

**Comment by ianegordon** [comment]
Z-Gesture works as expected to navigate back.  Please re-open if you have additional concerns.
